File: Modul 6 Sem 1/Phython/PRAK602-2410817110005-MuhammadRizqanKarim.py
# PRAK602-NIM-Nama.py
rooms = int(input("Masukkan jumlah ruangan: "))
zetsu = list(map(int, input("Masukkan jumlah zetsu awal: ").split()))

result = [zetsu[i] * (i + 1) for i in range(rooms)]
print(' '.join(map(str, result)))

File: Modul 6 Sem 1/C/PRAK602-2410817110005-MuhammadRizqanKarim.c
// PRAK602-NIM-Nama.c
#include <stdio.h>

int main() {
    int rooms;
    printf("Masukkan jumlah ruangan: ");
    scanf("%d", &rooms);
    int zetsu[rooms];
    printf("Masukkan jumlah zetsu awal: ");
    for (int i = 0; i < rooms; i++) {
        scanf("%d", &zetsu[i]);
    }
    for (int i = 0; i < rooms; i++) {
        printf("%d ", zetsu[i] * (i + 1));
    }
    printf("\n");
    return 0;
}
